.github/workflows/conda_build.yml

```
@@ -73,7 +79,7 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         run: sed -i "s+$PWD/++g" coverage.xml
       - name: SonarCloud Scan
-        if: matrix.os == 'ubuntu-latest'
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.python-version == '3.7' }}
```

Use latest Python for Sonarcloud
```suggestion
        if: ${{ matrix.os == 'ubuntu-latest' && matrix.python-version == '3.8' }}
```